Formula/libbs2b.rb

```diff
@@ -21,7 +21,18 @@ class Libbs2b < Formula
   depends_on "pkg-config" => :build
   depends_on "libsndfile"
 
+  on_macos do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   def install
+    if OS.mac?
+      inreplace "configure.ac", "dist-lzma", ""
```

Comment:
Why do we need this?

Reply:
Ah I forgot to commit my comment!  It's pretty simple - `autoconf` no longer allows `dist-lzma` in `configure.ac` and fails if this isn't removed.  I'll add the comment so this is more clear.